### ISSUE TITLE
Can't find integration in HACS

### ISSUE THREAD

**Issue by aviadlevy** [maorcc/citymind_water_meter#49]
Can't find the integration:
<img width="823" alt="image" src="https://user-images.githubusercontent.com/11833447/205237521-9fdfdc0a-4b56-467c-867f-a9197f470ee5.png">

Home assistant version: 
<img width="482" alt="image" src="https://user-images.githubusercontent.com/11833447/205237361-95e3fd8a-3902-4398-98b7-f37ff08d40f7.png">

Hacs version:
<img width="450" alt="image" src="https://user-images.githubusercontent.com/11833447/205237329-72708496-f831-442e-ac7a-93c204ad8742.png">

**Comment by elad-bar** [comment]
Did you install it via HACS, restart the HA and after refresh the integrations page and still not appear over there?
If yes, please post logs

Thanks

**Comment by aviadlevy** [comment]
Nope. Didn't install it yet. Just wanted to install for the first time but couldn't find the integration when searching hacs store

**Comment by elad-bar** [comment]
Just encountered the same problem, seems that repo does not contain the integration, as workaround you can set it as custom repo in hacs

**Comment by aviadlevy** [comment]
I'll give it a try.
Thanks!

**Comment by elad-bar** [comment]
Seems that repo was never added to HACS repo

**Comment by ShmulikMiata** [comment]
Hi,
Is there a solution for this?
Many thanks

**Comment by maorcc** [comment]
The goal is to make it an official Home Assistant integration.  @elad-bar and @OnFreund are working on it.

**Comment by elad-bar** [comment]
https://github.com/hacs/default/pull/2592
